ado2gh_migration_scripts/MigrationWithoutAssigningTeam/checkAfterMigration/checker.py

```diff
@@ -0,0 +1,22 @@
+import pandas as pd 
+import os 
+
+c_dir = os.path.dirname(os.path.realpath(__file__))
+## dependencies on the GetOrganizationRepositoriesFromGithub
+path = f'{c_dir}/orgRepositories.csv'
+migrated_df = pd.read_csv(path)
+migrated_df['full_name'] = migrated_df['full_name'].str.split('/').str[1]
+## dependencies on the migrationOfAzureReposToGithub
+path = f'{'/'.join(os.path.dirname(__file__).split('/')[0:-1])}/latestCommitInfoProjectRepoNames.csv'
```

Comment:
Did you run this code is this working? You are using single quote inside a single quote.

Reply:
Yeah It's Working Fine Because it's in the block of curly braces